README.md

```diff
@@ -8,40 +8,28 @@ This library helps convert [STAC Items](https://github.com/radiantearth/stac-spe
 
 ## Usage
 
-`stac_geoparquet.to_dataframe` does it all. You give it a list of (STAC Item) dictionaries. It just converts them to a `geopandas.GeoDataFrame`, which can be written to parquet with `.to_parquet`.
-
-```python
->>> import requests
->>> import stac_geoparquet
->>> item = requests.get("https://planetarycomputer.microsoft.com/api/stac/v1/collections/naip/items/ia_m_4209150_sw_15_060_20190828_20191105").json()
->>> df = stac_geoparquet.to_geodataframe([item])
->>> df.to_parquet("naip.parquet")
-```
+Use `stac_geoparquet.to_arrow.stac_items_to_arrow` and
+`stac_geoparquet.from_arrow.stac_table_to_items` to convert between STAC items
```

Comment:
I didn't give a lot of thought to the public API here. Maybe the public API should condense to `stac_geoparquet.arrow`? The function names are quite verbose here.

Reply:
Created https://github.com/stac-utils/stac-geoparquet/pull/47

Reply:
Updated after #47.